wpilibc/src/main/native/cpp/TimedRobot.cpp

```diff
@@ -32,8 +32,8 @@ void TimedRobot::StartCompetition() {
   // Loop forever, calling the appropriate mode-dependent function
   while (true) {
     // We don't have to check there's an element in the queue first because
-    // there's always at least one (the constructor adds one). It's reenqueued
-    // at the end of the loop.
+    // there's always at least one (the constructor adds one). It's re added to
+    // the queue at the end of the loop.
```

Comment:
https://english.stackexchange.com/a/93958

Reply:
regardless, it is still more confusing than just writing out what it means. There is no reason to use words that require googling to figure out unnecessarily.

Reply:
Re-enqueue is the proper CS term here, and it's commonly used in this context.

Reply:
I agree; I'd imagine most people would understand what enqueueing is in the context of queues, and so it's not a stretch to say that people would understand what re-enqueueing is.